packages/manager/modules/billing/src/billing.routing.js

```diff
@@ -25,6 +29,49 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       goToOrders: /* @ngInject */ ($state) => () =>
         $state.go('app.account.billing.orders'),
+
+      guides: /* @ngInject */ (currentUser) => {
+        return {
+          url: Object.fromEntries(
+            Object.entries(GUIDES_LIST).map(([key, value]) => {
+              return [
+                key,
+                Object.fromEntries(
+                  Object.entries(value).map(
+                    ([SecondLevelKey, SecondLevelValue]) => {
+                      return [
+                        SecondLevelKey,
+                        Object.fromEntries(
+                          Object.entries(SecondLevelValue).map(
+                            ([thirdLevelKey, thirdLevelValue]) => {
+                              return [
+                                thirdLevelKey,
+                                thirdLevelKey === 'url'
+                                  ? thirdLevelValue[
+                                      currentUser.ovhSubsidiary
+                                    ] || thirdLevelValue.DEFAULT
+                                  : thirdLevelValue,
+                              ];
+                            },
+                          ),
+                        ),
+                      ];
+                    },
+                  ),
+                ),
+              ];
+            }),
+          ),
+          tracking: GUIDE_TRACKING_TAG,
+        };
+      },
```

Comment:
**question:** Looks complex to understand, is it possible to split this?

Reply:
I tried something else to split into 3 functions.